icepyx/core/temporal.py

```diff
@@ -150,12 +158,10 @@ def validate_date_range_datestr(date_range, start_time=None, end_time=None):
 
     Parameters
     ----------
-    date_range: list(str)
+    date_range:
         A date range provided in the form of a list of strings
             Strings must be of formats accepted by validate_inputs_temporal.convert_string_to_date().
             List must be of length 2.
-    start_time: string, datetime.time, None
-    end_time:  string, datetime.time, None
```

Comment:
Why remove the `start_time` and `end_time` parameters? Shouldn't we just remove the type?

Reply:
Since they're part of the function signature, they will be documented automatically;  we only need to mention things in the docstring that aren't already represented in the function signature. One less thing to keep in sync :)